PendingReleaseNotes.md

```diff
@@ -20,6 +20,8 @@ v1.6...
 
 ### Ceph
 
+* Ceph Pacific support
+* Multiple Ceph Filesystems (with Pacific only)
```

Comment:
How about a comment about backward compatibility if they are using it?

Reply:
The intention is to hide the option, for whoever is using it nothing bad will happen, so I don't it's necessary to maintain it.